foodsaving/conversations/receivers.py

```diff
@@ -18,6 +23,31 @@ def mark_as_read(sender, instance, **kwargs):
     participant.save()
 
 
+# connect it once all notifications are completely implemented https://github.com/yunity/karrot-backend/issues/490
+# @receiver(post_save, sender=ConversationMessage)
+def notify_participants(sender, instance, **kwargs):
+    message = instance
+
+    if not message.conversation.target:
+        return
+
+    # exclude emails that had bounces or similar events recently
+    ignored_addresses = EmailEvent.objects.filter(created_at__gte=now() - relativedelta(months=6)).values('address')
+
+    participants_to_notify = ConversationParticipant.objects.filter(
+        conversation=message.conversation,
+        email_notifications=True
+    ).exclude(
+        user=message.author
+    ).exclude(
+        user__email__in=ignored_addresses
+    )
+
+    for participant in participants_to_notify:
+        with translation.override(participant.user.language):
```

Comment:
I would maybe put this in the `prepare_conversation_message_notification` function